lib/wanda_web/views/executions_view.ex

```diff
@@ -0,0 +1,7 @@
+defmodule WandaWeb.ExecutionsView do
+  use WandaWeb, :view
+
+  def render("list_checks_executions.json", %{data: data, total_count: total_count}) do
```

Comment:
Following `phoenix` convention, this function should be named `index`, but well, this name works as well.

Reply:
I meant the second variable hehe `index.json` hehe
And it should be named in the same name in the controller. But phoenix accepts any other naming in this case

Reply:
Yes, feel free to leave the current name if you prefer so